earthkit/data/readers/netcdf.py

```diff
@@ -628,7 +628,7 @@ def mutate_source(self):
 class NetCDFReader(Reader):
     def __init__(self, source, path):
         Reader.__init__(self, source, path)
-        NetCDFFieldList.__init__(self, path)
```

Comment:
`NetCDFFieldList` should be removed from this `__init__` since `NetCDFReader` has nothing to do with it!

Reply:
I should delete the line entirely?

Reply:
Yes, please.

Reply:
done